packages/checkout/widgets-lib/src/widgets/connect/ConnectWidgetRoot.tsx

```diff
@@ -37,6 +67,7 @@ export class Connect extends Base<WidgetType.CONNECT> {
               <ConnectWidget
                 config={this.strongConfig()}
                 checkout={this.checkout}
+                targetChainId={this.parameters.targetChainId}
```

Comment:
Nice one.. just double checking does this use the validated Params?

Reply:
Yes it does. The validated params stuff get calls in the constructor of BaseWidgetRoot. So this.parameters should be validated by there